android/build.gradle

```diff
@@ -61,5 +61,20 @@ allprojects {
         flatDir {
             dirs 'libs'
         }
+
+        exclusiveContent {
```

Comment:
why is this needed?

Reply:
When I build the app, I got 
```
* What went wrong:
Execution failed for task ':app:mergeDebugNativeLibs'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.MergeNativeLibsTask$MergeNativeLibsTaskWorkAction
   > 2 files found with path 'lib/arm64-v8a/libfbjni.so' from inputs:
      - /Users/linsang/.gradle/caches/transforms-3/4f41139e8a8b9fc7536f2fec86e6b63a/transformed/jetified-react-native-0.71.0-rc.0-debug/jni/arm64-v8a/libfbjni.so
      - /Users/linsang/.gradle/caches/transforms-3/84995423265cc7a30d31c99b7568043a/transformed/jetified-fbjni-0.3.0/jni/arm64-v8a/libfbjni.so
     If you are using jniLibs and CMake IMPORTED targets, see
     https://developer.android.com/r/tools/jniLibs-vs-imported-targets
```

Reply:
Do you have same issue if you use [`mavenCentral()`](https://github.com/aws-samples/amazon-chime-react-native-demo/pull/173/files#diff-197b190e4a3512994d2cebed8aff5479ff88e136b8cc7a4b148ec9c3945bd65aR58)?

Reply:
What do you mean? Thought we had mavenCentral defined in L49?